python/llm/example/GPU/PyTorch-Models/Model/llava/README.md

```diff
@@ -0,0 +1,79 @@
+# LLaVA
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on LLaVA models on [Intel GPUs](../README.md). For illustration purposes, we utilize the [liuhaotian/llava-v1.5-7b](https://huggingface.co/liuhaotian/llava-v1.5-7b) as a reference LLaVA model.
```

Comment:
As we have also verified llama 13b on Arc, we may use the format [here](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model/llama2) which includes both 7 and 13b models but use the 7b one as the default reference model.

Reply:
This may include minor changes in description, argument info, sample output, etc.